client/blocks/app-promo/index.jsx

```diff
@@ -1,55 +1,85 @@
 import { Card, Button } from '@automattic/components';
 import { useLocalizeUrl } from '@automattic/i18n-utils';
+import classNames from 'classnames';
 import { useTranslate, useRtl } from 'i18n-calypso';
 import wpToJpImageRtl from 'calypso/assets/images/jetpack/wp-to-jp-rtl.svg';
 import wpToJpImage from 'calypso/assets/images/jetpack/wp-to-jp.svg';
+import QrCode from 'calypso/blocks/app-promo/qr-code';
 import AppsBadge from 'calypso/blocks/get-apps/apps-badge';
 import CardHeading from 'calypso/components/card-heading';
 import userAgent from 'calypso/lib/user-agent';
-
+import { useSelector } from 'calypso/state';
+import { isUserLoggedIn } from 'calypso/state/current-user/selectors';
 import './style.scss';
 
-export const AppPromo = () => {
+export const AppPromo = ( {
+	title = false,
+	iconSize = 29,
+	source = 'calypso-customer-home',
+	subheader = false,
+	className = 'customer-home__card',
```

Comment:
That's a bit anti-pattern, it should have its own classname and append the passed classes to it. Like [this](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/button/index.tsx#L145),

Reply:
yeah that make more sense.